src/outline/dragDrop.js

```diff
@@ -1,441 +0,0 @@
-/**
```

Comment:
Was this code unused? If so, then yes please delete it! But we should make it a separate PR, since this is unrelated to the pad pane.